vmm/src/lib.rs

```diff
@@ -988,14 +989,26 @@ impl Vmm {
     fn create_vcpus(
         &mut self,
         entry_addr: GuestAddress,
+        request_ts: TimestampUs,
     ) -> std::result::Result<Vec<Vcpu>, StartMicrovmError> {
         let vcpu_count = self
             .vm_config
             .vcpu_count
             .ok_or(StartMicrovmError::VcpusNotConfigured)?;
         let mut vcpus = Vec::with_capacity(vcpu_count as usize);
+
+        // mmio_device_manager is instantiated in init_devices, which is called before
```

Comment:
nit: enclose mmio_device_manager, init_devices and start_vcpus in backtics

Reply:
done